internal/testing/pruning_test.go

```diff
@@ -190,12 +199,55 @@ func TestSectorPruning(t *testing.T) {
 		t.Fatal(err)
 	}
 
+	// create a contracts dict
+	c, err := b.Contracts(context.Background())
+	if err != nil {
+		t.Fatal(err)
+	}
+	contracts := make(map[types.PublicKey]api.ContractMetadata)
+	for _, contract := range c {
+		contracts[contract.HostKey] = contract
+	}
+
 	// add an object
 	if err := w.UploadObject(context.Background(), bytes.NewReader([]byte(t.Name())), "obj"); err != nil {
```

Comment:
I'd change this to adding multiple objects which each add 1 sector to the contract. That makes the proof verification a bit more meaningful. 

Later we will also need that when testing the actual pruning to verify the swap+trim operation works as expected. e.g. when we delete a sector from the middle of the contract we want to verify the relative positions of the other sectors remain the same within the contract.

Reply:
👍 good stuff, I rewrote the test to be more involved